client/src/commons/Map/Map.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+// @ts-ignore
+import GoogleMap from "google-map-react";
+import LocationInput, {
+    GeocodeResponse,
+    GoogleApiPlace
+} from "../LocationInputField/LocationInput";
+
+import useGoogleApiAutocomplete from "commons/LocationInputField/useGoogleApiAutocomplete";
+import injectScript from "commons/LocationInputField/useGoogleScriptInjector";
+import useGoogleGeocoder from "commons/LocationInputField/useGoogleGeocoder";
+import useDBParser from "Utils/vendor/useDBParsing";
+
+interface MapProps {
+    height?: number | string;
```

Comment:
I see that in the two times you're calling the map component you don't provide the height and width properties, so do you really need them? or there'll be in the future instances that we would want to provide our height/width?

Reply:
Also why number | string? I mean i get why string for px, vh, vw etc..

Reply:
number defaults to pxs

Reply:
its added so the container size could be customizable